bin/check-windows-process.rb

```diff
@@ -43,7 +43,7 @@
   opts.on('-p', '--processname PROCESS', 'Unique process string to search for.') do |p|
     options[:procname] = p
     if p == ''
-      STDERR.puts 'Empty string for -p : Expected a string to match against.'
+      warn 'Empty string for -p : Expected a string to match against.'
```

Comment:
as you can see below it exits a 3 so this should be an unknown instead to maintain backwards compatibility.